wallet/wallet.c

```diff
@@ -1409,15 +1403,14 @@ bool wallet_init_channels(struct wallet *w)
 	return wallet_channels_load_active(w);
 }
 
-static enum channel_state_db get_state_channel_db(const char *dir, const char *typ)
+static enum channel_state_bucket get_state_channel_db(const char *dir, const char *typ)
```

Comment:
nice! this should be folded in to the original commit at some point before we merge this into master, so that the original commit has the complete/perfect changeset ;)